c7n/cli.py

```diff
@@ -297,8 +295,23 @@ def setup_parser():
     # access.add_argument(
     #    '-m', '--access', default=False, action='store_true')
 
-    run_desc = ("Execute the policies in a config file")
-    run = subs.add_parser("run", description=run_desc, help=run_desc)
+    run_desc = "\n".join((
+        "Execute the policies in a config file",
+        "",
+        "Multiple regions can be passed in, as can the symbolic region 'all'. ",
+        "",
+        "When running across multiple regions, policies targeting resources in ",
+        "regions where they do not exist will not be run. The output directory ",
```

Comment:
yup thats the goal. its almost the same behavior between single vs multi region on non existent with one minor diff, it does trigger a warning log level message when running single region vs running multi region on resource not present in region. in contrast current release would generate a traceback/stacktrace in that case (and a exit 1).

```
(custodian) bash-3.2$ custodian run --region all mu.yml  -s out -f r.db
2017-05-15 06:07:41,356: c7n.policies:WARNING policy:all-lambdas resources:lambda not available in region:ca-central-1
Empty policy file(s).  Nothing to do.
(custodian) bash-3.2$ custodian run --region all mu.yml  -s out -f r.db
2017-05-15 06:07:57,746: custodian.policy:INFO policy: all-lambdas resource:lambda region:ap-northeast-1 count:0 time:0.98
2017-05-15 06:07:58,942: custodian.policy:INFO policy: all-lambdas resource:lambda region:ap-northeast-2 count:0 time:1.19
2017-05-15 06:08:00,612: custodian.policy:INFO policy: all-lambdas resource:lambda region:ap-south-1 count:0 time:1.66
2017-05-15 06:08:01,941: custodian.policy:INFO policy: all-lambdas resource:lambda region:ap-southeast-1 count:0 time:1.32
2017-05-15 06:08:03,067: custodian.policy:INFO policy: all-lambdas resource:lambda region:ap-southeast-2 count:0 time:1.12
2017-05-15 06:08:03,712: custodian.policy:INFO policy: all-lambdas resource:lambda region:eu-central-1 count:0 time:0.64
2017-05-15 06:08:04,397: custodian.policy:INFO policy: all-lambdas resource:lambda region:eu-west-1 count:0 time:0.68
2017-05-15 06:08:04,927: custodian.policy:INFO policy: all-lambdas resource:lambda region:eu-west-2 count:0 time:0.52
2017-05-15 06:08:06,085: custodian.policy:INFO policy: all-lambdas resource:lambda region:us-east-1 count:15 time:1.15
2017-05-15 06:08:06,974: custodian.policy:INFO policy: all-lambdas resource:lambda region:us-east-2 count:6 time:0.88
2017-05-15 06:08:07,818: custodian.policy:INFO policy: all-lambdas resource:lambda region:us-west-1 count:1 time:0.84
2017-05-15 06:08:08,409: custodian.policy:INFO policy: all-lambdas resource:lambda region:us-west-2 count:0 time:0.58```